platform/tests/application_test.go

```diff
@@ -0,0 +1,92 @@
+package tests
+
+import (
+	"context"
+	"testing"
+
+	"github.com/GuiaBolso/darwin"
+	"github.com/fewlinesco/go-pkg/platform"
+)
+
```

Comment:
Maybe we should add tests somewhere else? We only add utility functions for testing in this package before, it seems weird to me that we're adding actual tests for `go-pkg` in there. We should separate `go-pkg` integration tests like this one from the test utilities we want to have in `go-pkg`.
Maybe rename it `test_utilities` and `integration_tests`so that no confusion is possible ?

Reply:
I would definitely change the `pkg` name to `platform_tests` as this is the convention in Go, but since this also tests the applictation setup it seems more like an integration test so leaving it directly in the `platform` folder it's fine for me.  If it were unit tests for the `connect` function I'd move it but I don't think this is necessary in this case.